styles/index.css

```diff
@@ -0,0 +1,1038 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+p {
+  font-family: 'Lato', sans-serif;
+}
+
+body {
+  margin: 0;
+  top: 0;
+  left: 0;
+  padding: 0;
+}
+
+main {
+  height: 100%;
+  margin: 0;
+  width: 320px;
+}
+
+.navbar-desktop {
+  background-image: url('/images/safari-white.png');
+  width: 320px;
+  height: 70px;
+  position: fixed;
+  right: 0;
+  left: 0;
+  top: 0;
+  z-index: 9999;
+}
+
+.social-media-icons {
+  display: none;
+  height: 20px;
+}
+
+.social-media-icons-ul {
+  display: none;
+  flex-direction: row;
+  list-style: none;
+  margin-top: 0;
+  justify-content: flex-end;
+  margin-right: 50px;
+}
+
+.fa-facebook-f {
+  color: white;
+  margin: 8px;
+}
+
+.fa-twitter {
+  color: white;
+  margin: 8px;
+}
+
+.nav-links-ul {
+  display: flex;
+  list-style-type: none;
+  height: 100%;
+  background-color: #ec5242;
+  align-items: center;
+  align-content: center;
+  position: fixed;
+  left: -100%;
+  top: 70px;
+  gap: 0;
+  flex-direction: column;
+
+  /* background-color: #ec5242; */
+  width: 320px;
+  transition: 0.3s;
+}
+
+.nav-links {
+  top: 70px;
+  padding: 10px;
+  padding: 20px;
+  width: 100%;
+  align-content: center;
+}
+
+.nav-links:hover {
+  background-color: #ff8d8d;
+}
+
+.nav-links a {
+  display: flex;
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  text-decoration: none;
+  width: 100%;
+  align-items: center;
+  color: white;
+}
+
+.logo {
+  display: none;
+  text-decoration: none;
+
+  /* vertical-align: center; */
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+  margin-top: 20px;
+}
+
+.bar {
+  display: block;
+  width: 30px;
+  height: 4px;
+  margin: 7px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: 0.3s ease-in-out;
+  background-color: black;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(12px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-10px) rotate(-45deg);
+}
+
+.nav-links-ul.active {
+  left: 0;
+}
+
+@media (min-width: 320px) {
+  main {
+    width: 100%;
+  }
+
+  .navbar-desktop {
+    width: 100%;
+  }
+
+  .nav-links-ul {
+    width: 100%;
+  }
+}
+
+@media (min-width: 769px) {
+  .hamburger {
+    display: none;
+  }
+
+  .navbar-desktop {
+    background-image: none;
+    background-color: white;
+    height: 100px;
+    position: fixed;
+  }
```

Comment:
- [ ] Please kindly align your menu in the correct way, the logo is supposed to be to the left, and the nav-link to the right side.

Your design
<img width="769" alt="Screen Shot 2023-01-12 at 13 56 10" src="https://user-images.githubusercontent.com/74501523/212049139-92669a05-7700-4c06-9235-5230551bd125.png">

Original design
<img width="1073" alt="Screen Shot 2023-01-12 at 13 56 40" src="https://user-images.githubusercontent.com/74501523/212049266-1f9fa1f9-b2ee-44e5-986d-162004f3135f.png">

- [ ] Also when I reduce the size of the screen at some point the logo disappears 🫠 🫥. Your nav-menu should be responsiveness, even user reducer the size od the screen the logo should be on the left and the link to the right side 👍 
<img width="637" alt="Screen Shot 2023-01-12 at 13 57 13" src="https://user-images.githubusercontent.com/74501523/212049557-3480b28d-2017-4a2f-9b59-f0c84132ba60.png">